src/cdk/datepicker/datepicker-intl.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs';
+
+
+/** Datepicker data that requires internationalization. */
+@Injectable({providedIn: 'root'})
+export class CdkDatepickerIntl {
+    /**
+     * Stream that emits whenever the labels here are changed. Use this to notify
+     * components if the labels have changed after initialization.
+     */
+    readonly changes: Subject<void> = new Subject<void>();
+
+    /** A label for the previous month button (used by screen readers). */
+    prevMonthLabel: string = 'Previous month';
```

Comment:
Are we going to use these labels in cdk without templates? If not it's better to keep it in material.

Reply:
I was thinking that these labels in the cdk would be useful so that users that weren't using the material calendar could use the cdk to add their own labels to their own material calendar buttons. These labels are also universal in the sense that each datepicker should have a prev/next label for month, year, and multi-years. Do you know of a better way to accomplish this?